_posts/2022-03-16-launch-announcement-1-3-0.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title:  "OpenSearch 1.3.0 is out now!"
+authors:
+  - andhopp
+  - jadhanir
+  - jdbright
+  - ylwu
+date: 2022-03-16 08:50:01
+categories:
+  - releases
+twittercard:
+    description: "With this latest version of the OpenSearch distribution (OpenSearch, OpenSearch Dashboards, as well as plugins and tools) you can enjoy a number of new features and enhancements as well as improvements to stability and efficiency."
+---
+
+With this latest version of the OpenSearch distribution (OpenSearch, OpenSearch Dashboards, as well as plugins and tools) you can enjoy a number of new features and enhancements as well as improvements to stability and efficiency.
+
+One of the major new additions to 1.3.0 is a set of tools for observability between logs, metrics, and other live data. When critical, event-based information is correlated with other application or system data and reported centrally, availability issues are easier to identify and resolve. Try these new tools out (see Observability, below) and let the community know what you think.
+
+Here is a list of major and minor feature enhancements that have been included in this release:
+
+### Index State Management
+
+* **Continuous Mode in Transform**: With this release, you can now run transforms in a continuous mode based on a schedule. This enables you to keep an incremental transform job running on a group of source indices actively ingesting new data. At each run, only the modified buckets will be transformed, reducing the overhead for users who want to their transform up to date.
+
+### Observability
+
+Customers can now create custom Observability Applications to view the availability status of all their systems, where log events can be combined with trace and metric data, into a single view of system health empowering developers and IT Ops to resolve issues faster and with fewer escalations.
```

Comment:
'Customers' is weird in an open source blog. "Users" maybe?

Reply:
Moved to Users